heidrun/getty.rb

```diff
@@ -0,0 +1,134 @@
+LANGUAGE_MAP = {
+  'multiple languages' => 'mul'
+}
+
+DATA_PROVIDER_MAP = {
+  'GETTY_ROSETTA' => 'Getty Research Institute',
+  'GETTY_OCP' => 'Getty Research Institute'
+}
+
+build_url = lambda do |p|
+  source = p.root[Krikri::PrimoParser.record('control', 'sourceid').join('/')]
+           .first.value
+  id = p.root[Krikri::PrimoParser.record('control', 'recordid').join('/')]
+       .first.value
+
+  base_url = 'http://primo.getty.edu/primo_library/libweb/action/dlDisplay.do'
+
+  case source
+  when 'GETTY_ROSETTA'
+    "#{base_url}?vid=GRI&afterPDS=true&institution=01GRI&docId=#{id}"
+  when 'GETTY_OCP'
+    "#{base_url}?vid=GRI-OCP&afterPDS=true&institution=01GRI&docId=#{id}"
+  end
+end
+
+lookup_language = lambda do |i|
```

Comment:
I think we should be fine to just map the language string directly into `providedLabel`. Both cases here should be handled by [LanguageToLexvo](https://github.com/dpla/KriKri/blob/develop/lib/krikri/enrichments/language_to_lexvo.rb). For the 639-2 codes, those should all be present in 639-3. `"multiple languages"` is likewise present as a label, and will get treated as the language label and linked to `lexvo639_3:mul`, as though it were a real language's name.